contracts/protocol/libraries/logic/LiquidationLogic.sol

```diff
@@ -831,4 +855,27 @@ library LiquidationLogic {
             );
         }
     }
+
+    /**
+     * @notice Convert msg.value to WETH and check if liquidationAsset is WETH (if msg.value > 0)
```

Comment:
should it means: Convert msg.value to WETH and check if liquidationAsset is `ETH` (if msg.value > 0) ?